atomic_reactor/utils/operator.py

```diff
@@ -162,6 +338,12 @@ class NotOperatorCSV(Exception):
     """
 
 
+class ValidationError(Exception):
+    """
+    CSV validation against schema failed
+    """
```

Comment:
Where is this error raised?

And looks this error is specific to a failure CSV validation, rename it to a more specific name? `ValidationError` is too general.

Reply:
@tkdchen; L# 799 in operator.py

Good point about the name.